Configuring Node-RED dashboard 2.0 on Home Assistant

omrid01 opened the pull request:
## Description

<!-- Describe your changes in detail -->

## Related Issue(s)

<!-- What issue does this PR relate to? -->

## Checklist

<!-- https://flowfuse.com/handbook/development/#defining-done -->

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] Suitable unit/system level tests have been added and they pass <!-- If not adding test coverage, please clarify why not? -->
 - [ ] Documentation has been updated
    - [ ] Upgrade instructions
    - [ ] Configuration details
    - [ ] Concepts
 - [ ] Changes `flowforge.yml`?
    - [ ] Issue/PR raised on `FlowFuse/helm` to update ConfigMap Template
    - [ ] Issue/PR raised on `FlowFuse/CloudProject` to update values for Staging/Production
 - [ ] Link to Changelog Entry PR, or note why one is not needed.

## Labels

 - [ ] Includes a DB migration? -> add the `area:migration` label

Steve-Mcl commented on docs/en/user/home-assistant.md:
```diff
@@ -0,0 +1,29 @@
+2. Create a new iframe container for hosting the FlowFuse Dashboard clients:
+- In HA, go to **Settings->Dashboards->Add dashboard**, select **Webpage**
+
+![Image](https://github.com/user-attachments/assets/6b6323f1-e3a3-41d2-8893-c4a79a2cb341)
```
Please add the image to `docs/assets/images` and point this to it like this:
```md
![A note to what this image is](../../../assets/images/my-image-file-name.png){data-zoomable}
*A note to what this image is*
````

Steve-Mcl commented on docs/en/user/home-assistant.md:
```diff
@@ -0,0 +1,29 @@
+
+- Set the URL of the dashboard, e.g. `<HA host>:1880/endpoint/dashboard`
+
+![Image](https://github.com/user-attachments/assets/9815d378-ca6c-458c-8de4-33b40934bd05)
```
Please add the image to `docs/assets/images` and point this to it like this:
```md
![A note to what this image is](../../../assets/images/my-image-file-name.png){data-zoomable}
*A note to what this image is*
````

Steve-Mcl commented on docs/en/user/home-assistant.md:
```diff
@@ -0,0 +1,29 @@
+![Image](https://github.com/user-attachments/assets/9815d378-ca6c-458c-8de4-33b40934bd05)
+- Set the iframe a title & optional icon (which will show in the HA sidebar), and click **Create**
+
+![Image](https://github.com/user-attachments/assets/38428345-cac0-4be7-b1fc-33c741c2db95)
```
Please add the image to `docs/assets/images` and point this to it like this:
```md
![A note to what this image is](../../../assets/images/my-image-file-name.png){data-zoomable}
*A note to what this image is*
````

Steve-Mcl commented on docs/en/user/home-assistant.md:
```diff
@@ -0,0 +1,29 @@
+
+- You can now open the FlowFuse Dashboard from the HA sidebar, or directly via the endpoint URL defined above
+
+![Image](https://github.com/user-attachments/assets/45beadf3-bfe1-45da-83c8-561191387d2e)
```
Please add the image to `docs/assets/images` and point this to it like this:
```md
![A note to what this image is](../../../assets/images/my-image-file-name.png){data-zoomable}
*A note to what this image is*
````

Steve-Mcl reviewed (commented):
Hi @omrid01 - I would like to get this merged. Would you be kind enough to address the issues raised in review please?

omrid01 commented:
Hi Steve, Joe,
Sorry, I was away on an urgent personal matter. Will address this today or
tomorrow.
Omri

On Sun, Oct 5, 2025 at 12:06 PM Stephen McLaughlin ***@***.***>
wrote:

> ***@***.**** commented on this pull request.
>
> Hi @omrid01 <https://github.com/omrid01> - I would like to get this
> merged. Would you be kind enough to address the issues raised in review
> please?
> ------------------------------
>
> In docs/en/user/home-assistant.md
> <https://github.com/FlowFuse/node-red-dashboard/pull/1805#discussion_r2404372675>
> :
>
> > +Node-RED can be integrated with Home Assistant (**HA**) in 2 ways:
> +- Running Node-RED on an external host, and connecting to the HA server via HTTP messaging or the `node-red-contrib-home-assistant-websocket` node-set
> +- Using the internal Node-RED add-on within HA, which by default is part of the HA installation
> +
> +### Configuring Node-RED dashboard 2.0 in Home Assistant
> +
> +As of today, the Node-RED add-on within HA includes the Node-RED dashboard 1.0 nodes (`node-red-dashboard`), and exposes its URL (`.../ui`). However, Node-RED Dashboard 1.0 is now deprecated and replaced has been replaced by FlowFuse Dashboard.
> +
> +Following are instructions for installing & configuring FlowFuse Dashboard into the Node-RED add-on in HA
> +
> +1. Within the Node-RED add-on, install the FlowFuse Dashboard nodes ***@***.***/node-red-dashboard`) via the "Manage Palette" option.
> +
> +2. Create a new iframe container for hosting the FlowFuse Dashboard clients:
> +- In HA, go to **Settings->Dashboards->Add dashboard**, select **Webpage**
> +
> +![Image](https://github.com/user-attachments/assets/6b6323f1-e3a3-41d2-8893-c4a79a2cb341)
>
> Please add the image to docs/assets/images and point this to it like this:
>
> ![A note to what this image is](../../../assets/images/my-image-file-name.png){data-zoomable}*A note to what this image is*
>
> ------------------------------
>
> In docs/en/user/home-assistant.md
> <https://github.com/FlowFuse/node-red-dashboard/pull/1805#discussion_r2404372738>
> :
>
> > +### Configuring Node-RED dashboard 2.0 in Home Assistant
> +
> +As of today, the Node-RED add-on within HA includes the Node-RED dashboard 1.0 nodes (`node-red-dashboard`), and exposes its URL (`.../ui`). However, Node-RED Dashboard 1.0 is now deprecated and replaced has been replaced by FlowFuse Dashboard.
> +
> +Following are instructions for installing & configuring FlowFuse Dashboard into the Node-RED add-on in HA
> +
> +1. Within the Node-RED add-on, install the FlowFuse Dashboard nodes ***@***.***/node-red-dashboard`) via the "Manage Palette" option.
> +
> +2. Create a new iframe container for hosting the FlowFuse Dashboard clients:
> +- In HA, go to **Settings->Dashboards->Add dashboard**, select **Webpage**
> +
> +![Image](https://github.com/user-attachments/assets/6b6323f1-e3a3-41d2-8893-c4a79a2cb341)
> +
> +- Set the URL of the dashboard, e.g. `<HA host>:1880/endpoint/dashboard`
> +
> +![Image](https://github.com/user-attachments/assets/9815d378-ca6c-458c-8de4-33b40934bd05)
>
> Please add the image to docs/assets/images and point this to it like this:
>
> ![A note to what this image is](../../../assets/images/my-image-file-name.png){data-zoomable}*A note to what this image is*
>
> ------------------------------
>
> In docs/en/user/home-assistant.md
> <https://github.com/FlowFuse/node-red-dashboard/pull/1805#discussion_r2404372752>
> :
>
> > +
> +Following are instructions for installing & configuring FlowFuse Dashboard into the Node-RED add-on in HA
> +
> +1. Within the Node-RED add-on, install the FlowFuse Dashboard nodes ***@***.***/node-red-dashboard`) via the "Manage Palette" option.
> +
> +2. Create a new iframe container for hosting the FlowFuse Dashboard clients:
> +- In HA, go to **Settings->Dashboards->Add dashboard**, select **Webpage**
> +
> +![Image](https://github.com/user-attachments/assets/6b6323f1-e3a3-41d2-8893-c4a79a2cb341)
> +
> +- Set the URL of the dashboard, e.g. `<HA host>:1880/endpoint/dashboard`
> +
> +![Image](https://github.com/user-attachments/assets/9815d378-ca6c-458c-8de4-33b40934bd05)
> +- Set the iframe a title & optional icon (which will show in the HA sidebar), and click **Create**
> +
> +![Image](https://github.com/user-attachments/assets/38428345-cac0-4be7-b1fc-33c741c2db95)
>
> Please add the image to docs/assets/images and point this to it like this:
>
> ![A note to what this image is](../../../assets/images/my-image-file-name.png){data-zoomable}*A note to what this image is*
>
> ------------------------------
>
> In docs/en/user/home-assistant.md
> <https://github.com/FlowFuse/node-red-dashboard/pull/1805#discussion_r2404372776>
> :
>
> > +
> +2. Create a new iframe container for hosting the FlowFuse Dashboard clients:
> +- In HA, go to **Settings->Dashboards->Add dashboard**, select **Webpage**
> +
> +![Image](https://github.com/user-attachments/assets/6b6323f1-e3a3-41d2-8893-c4a79a2cb341)
> +
> +- Set the URL of the dashboard, e.g. `<HA host>:1880/endpoint/dashboard`
> +
> +![Image](https://github.com/user-attachments/assets/9815d378-ca6c-458c-8de4-33b40934bd05)
> +- Set the iframe a title & optional icon (which will show in the HA sidebar), and click **Create**
> +
> +![Image](https://github.com/user-attachments/assets/38428345-cac0-4be7-b1fc-33c741c2db95)
> +
> +- You can now open the FlowFuse Dashboard from the HA sidebar, or directly via the endpoint URL defined above
> +
> +![Image](https://github.com/user-attachments/assets/45beadf3-bfe1-45da-83c8-561191387d2e)
>
> Please add the image to docs/assets/images and point this to it like this:
>
> ![A note to what this image is](../../../assets/images/my-image-file-name.png){data-zoomable}*A note to what this image is*
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/FlowFuse/node-red-dashboard/pull/1805#pullrequestreview-3302291828>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AX3RLF4SVAJ7K4AXEC2RQE33WDNQZAVCNFSM6AAAAACEVT3F7SVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZTGMBSGI4TCOBSHA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

omrid01 commented:
I have updated the document with locally placed images. Is there anything else I need to do, or do the updates in the fork automatically propagate into the Pull Request?

Steve-Mcl reviewed (approved):
LGTM. 
Thank you very much for the docs update.
